sentry/src/main/java/io/sentry/config/FilesystemPropertiesLoader.java

```diff
@@ -0,0 +1,41 @@
+package io.sentry.config;
+
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/** Loads {@link Properties} from file system location. */
+final class FilesystemPropertiesLoader implements PropertiesLoader {
+  private final @NotNull String filePath;
+  private final @NotNull ILogger logger;
+
+  public FilesystemPropertiesLoader(@NotNull String filePath, @NotNull ILogger logger) {
+    this.filePath = filePath;
+    this.logger = logger;
+  }
+
+  @Override
+  public @Nullable Properties load() {
+    try {
+      final File f = new File(filePath);
+      if (f.isFile() && f.canRead()) {
+        try (InputStream is = new FileInputStream(f)) {
```

Comment:
is it worth to wrap up on a buffered type?

Reply:
https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/OutboxSender.java#L59

Reply:
Changed.